addons/website_slides/tests/test_slide_channel.py

```diff
@@ -121,6 +121,19 @@ def test_channel_partner_next_slide(self):
             # we have gone through all the content, next slide should be False
             self.assertFalse(membership.next_slide_id)
 
+    @users('user_officer')
+    def test_copy_slide_channel_enroll(self):
```

Comment:
Initially I though "Could we perhaps move this test below the last existing one as it targets the same feature?"
```suggestion
    def test_channel_enroll_policy_copy_with_visibility_member(self):
```
But actually, this could be included in the existing test, which is short and ends up in your setup state doesn't it?